### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by teobugslayer**
Refs https://github.com/NativeScript/nativescript/issues/488

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,72 +24,72 @@
     "mobile"
   ],
   "dependencies": {
-    "bplist-parser": "0.0.6",
-    "broccoli": "0.16.2",
+    "bplist-parser": "0.1.0",
```
Comment (Fatme) [inline package.json]: Have you tested if this update is fine?

**Comment by dtopuzov** [comment]
:+1:

File: package.json [hunk 2]
```diff
@@ -24,72 +24,72 @@
...
+    "broccoli": "0.16.4",
     "broccoli-funnel": "0.2.3",
     "bufferpack": "0.0.6",
     "bufferutil": "https://github.com/telerik/bufferutil/tarball/master",
-    "byline": "4.1.1",
-    "chalk": "1.0.0",
+    "byline": "4.2.1",
+    "chalk": "1.1.0",
     "cli-table": "https://github.com/telerik/cli-table/tarball/master",
```
Comment (rosen-vladimirov) [inline package.json]: we should not target master, can you update this dependency as well, the target should be: 
`"https://github.com/telerik/cli-table/tarball/v0.3.1.1",`

File: package.json [hunk 3]
```diff
@@ -24,72 +24,72 @@
...
     "tabtab": "https://github.com/Icenium/node-tabtab/tarball/master",
-    "temp": "0.8.1",
+    "temp": "0.8.3",
     "through2": "2.0.0",
     "utf-8-validate": "https://github.com/telerik/utf-8-validate/tarball/master",
-    "vinyl-filter-since": "2.0.0",
+    "vinyl-filter-since": "2.0.2",
     "winreg": "0.0.12",
-    "ws": "0.7.1",
+    "ws": "0.7.2",
```
Comment (rosen-vladimirov) [inline package.json]: https://github.com/websockets/ws/commit/046390f92b0dd59d30664a5be368b5ccb8217314
Due to this change in ws, we have to update our forks of `utf-8-validate` and `bufferutil`

**Comment by rosen-vladimirov** [comment]
:+1:  merge, so we'll have enough time for testing :P

**Pull request merged**